src/libsyntax/ext/tt/quoted.rs

```diff
@@ -396,11 +418,11 @@ fn parse_sep_and_kleene_op(
                 );
 
                 // Return a dummy
-                return (None, KleeneOp::ZeroOrMore);
+                return (None, KleeneToken { span: DUMMY_SP, op: KleeneOp::ZeroOrMore });
```

Comment:
The correct span from `parse_kleene_op` is available here
Using some real spans if that's possible is preferable to using `DUMMY_SP`, even if those spans are approximate.

Reply:
Oops. I went a bit fast on that part. Thanks for noticing!